mapswipe_workers/mapswipe_workers/project_types/arbitrary_geometry/grouping_functions.py

```diff
@@ -65,50 +61,22 @@ def group_input_geometries(input_geometries_file, group_size):
         try:
             groups[group_id_string]
         except KeyError:
-            # todo: change this and call feature_geoms -> geojson
-            groups[group_id_string] = {
-                "feature_ids": [],
-                "feature_geometries": [],
-                "center_points": [],
-                "reference": [],
-                "screen": [],
-            }
+            groups[group_id_string] = {"feature_ids": [], "feature": []}
```

Comment:
to be strict we would use `features` here, right? as this will be a list of more than one feature.